packages/components/.babelrc

```diff
@@ -0,0 +1,23 @@
+{
+  "presets": [
+    ["@babel/preset-env", {"modules": false}],
```

Comment:
modules should be probably 'auto' no?

Reply:
https://babeljs.io/docs/en/babel-preset-env#modules

False means: Keep ESM module syntax

Otherwise babel would transform all `import blubb from 'blabb'` to `requre(...)` -> no treeshaking

Reply:
i thought it adjusts for rollup automagically